build_artifacts/v1/v1.0/v1.0.0/v1.0.0-beta/Dockerfile

```diff
@@ -24,11 +25,13 @@ RUN usermod "--login=${NB_USER}" "--home=/home/${NB_USER}" --move-home "-u ${NB_
 ENV MAMBA_USER=$NB_USER
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends sudo gettext-base wget curl unzip git rsync && \
+    apt-get install -y --no-install-recommends sudo gettext-base wget curl unzip git rsync build-essential && \
     # We just install tzdata below but leave default time zone as UTC. This helps packages like Pandas to function correctly.
     DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends awscli tzdata && \
     chmod g+w /etc/passwd && \
     echo "ALL    ALL=(ALL)    NOPASSWD:    ALL" >> /etc/sudoers && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends krb5-user && \
```

Comment:
Is there any reason we can't append `krb5-user` to the previous installation line where awscli and tzdata are installed?

Reply:
I wanted to keep it separate as it's doing a specific thing of adding a Kerberos user and setting up permissions for it.

Reply:
Sure I can do that it's just easier to manage if we want to remove dependency.